src/models/subscription.ts

```diff
@@ -39,7 +39,7 @@ export interface RepositoryData {
 	[key: string]: unknown;
 }
 
-export type TaskStatus = "pending" | "complete";
+export type TaskStatus = "pending" | "complete" | "failed";
```

Comment:
Currently, a task can either be "pending" (i.e. we're working on it) or "complete". Introducing the state "failed" to mark that a task has failed for some reason.